decentraland/components/presentational/MyAsset.js

```diff
@@ -32,8 +31,41 @@ MyAsset.propTypes = {
   asset: PropTypes.object.isRequired,
 };
 
+export function MyAssetInfo({ asset }) {
```

Comment:
Where is this component used outside of a test?

Reply:
Nowhere

Reply:
Hmm, I'm confused. Can you please explain more?

Reply:
`MyAssetInfo` component is being used only for the `MyAsset` component and its test.

Reply:
But before this recent change https://github.com/tasitlabs/tasit/pull/307/files/2f785b92e176c5538b8318c4f756bee4b8109c65..d963eb5f89b72283a834603a77a307d5106619c6#r276674572 it wasn't being used anywhere.

Now it makes sense.

Reply:
Feel free to resolve this after reading

Reply:
Oh, yes, I've forgotten to render after refactor it.
Resolving this.